service_overview.py

```diff
@@ -42,11 +42,6 @@ class Service(Enum):
         "link": "https://github.com/code-kern-ai/refinery-authorizer",
         "public_repo": True,
     },
-    Service.CONFIG: {
```

Comment:
should also be removed from enum & a note for the removal of the entry in the database table for the release (dev & prod) similar to when we removed zero shot.